### PR TITLE
Added vendor_billing_id field in the License entity

### PR DESCRIPTION
**Pull request by pfeyssaguet**

### CODE REVIEW

File: doc/licenses.md [hunk 1]
```diff
@@ -148,36 +149,36 @@ Its purpose is to present some fields of the offer linked to the license, update
...
 
-| Field | Type     | Example   | Description       |
-| ----- | -------- | --------- | ----------------- |
-| cycle | `string` | 720       | The billing cycle |
-| term  | `string` | 8640      | The term          |
-| type  | `string` | RECURRING | The pricing type  |
+| Field | Type     | Example   | Description            |
+|-------|----------|-----------|------------------------|
+| cycle | `string` | 720       | The billing cycle      |
+| term  | `string` | 8640      | The term         <br/> |
```
Comment (Dhalsim404) [inline doc/licenses.md:155]: <br/> ?

File: src/Licenses/Entities/License/License.php [hunk 2]
```diff
@@ -733,6 +749,7 @@ public function jsonSerialize(): array
             self::COLUMN_MARKETPLACE            => $this->marketplace,
             self::COLUMN_ACTIVE_SEATS           => $this->activeSeats->jsonSerialize(),
             self::COLUMN_FRIENDLY_NAME          => $this->friendlyName,
+            self::COLUMN_VENDOR_BILLING_ID => $this->vendorBillingId,
```
Comment (Dhalsim404) [inline src/Licenses/Entities/License/License.php:752]: I think it is better to pad with space (or either remove padding on every array elements)

**Comment by sabirab2** [comment]
A new constant should be added to `LicenseFindFieldEnum` class as below :

```php
LicenseFindFieldEnum::OFFER_VENDOR_BILLING_ID   => LicenseOfferFindResult::COLUMN_LICENSE . '.' . License::COLUMN_VENDOR_BILLING_ID
```

https://github.com/pfeyssaguet/public-api-client/blob/vendor-billing-id/src/Licenses/Enum/LicenseFindFieldEnum.php#L185

File: doc/licenses.md (hunk 1 above)
Comment (pfeyssaguet) [reply doc/licenses.md:155]: fixed

File: src/Licenses/Entities/License/License.php (hunk 2 above)
Comment (pfeyssaguet) [reply src/Licenses/Entities/License/License.php:752]: fixed

**Review by sabirab2 — APPROVED** [review]

**Pull request merged**